### ISSUE TITLE
[rollup-plugin-workbox] recently upgraded dependency on pretty-bytes fails builds

### ISSUE THREAD

**Issue by dlockhart** [modernweb-dev/web#2212]
The [recent upgrade of pretty-bytes](https://github.com/modernweb-dev/web/pull/2208) from version 5.6.0 to 6.1.0 which was released in `rollup-plugin-workbox` version 6.2.1 is causing Rollup builds using `@open-wc/building-rollup` to fail with the following error:

```
Error: While loading the Rollup configuration from "rollup.config.js", Node tried to require an ES module from a CommonJS file, which is not supported. A common cause is if there is a package.json file with "type": "module" in the same folder. You can try to fix this by changing the extension of your configuration file to ".cjs" or ".mjs" depending on the content, which will prevent Rollup from trying to preprocess the file but rather hand it to Node directly.
```

It seems related to the fact that `pretty-bytes` [upgraded to be ESM-based](https://github.com/sindresorhus/pretty-bytes/compare/v5.6.0...v6.1.0) and now has `"type": "module"`. The way in which it gets imported into `rollup-plugin-workbox` may need to be adjusted.

## Workaround

You can work around this by pinning the version of `pretty-bytes` in your `package.json`:

```json
{
  "overrides": {
    "pretty-bytes": "5.6.0"
  }
}
```

**Comment by thepassle** [comment]
Would you mind creating a pr to fix it? :)

**Comment by dlockhart** [comment]
Fixed in #2213.
